src/lib/liquidiumServer.test.ts

```
@@ -0,0 +1,430 @@
+            'Custom-Header': 'custom-value',
+          },
+        },
+      );
+    });
+  });
+
+  describe('error responses', () => {
+    it('returns error when API key is missing', async () => {
+      // Temporarily remove API key
+      const originalApiKey = process.env.LIQUIDIUM_API_KEY;
+      delete process.env.LIQUIDIUM_API_KEY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)